test/new/db/cmd/cmd_macros

```diff
@@ -89,19 +89,19 @@ EXPECT=<<EOF
 EOF
 CMDS=<<EOF
 wx 010203040506070809
-3(foo,p8 4,s+$0)(1)
+3(foo;p8 4;s+$0)(1)
 EOF
 RUN
 
-NAME=3(foo,?e a,?e b)()
+NAME=3(foo;?e a;?e b)()
 FILE=-
 EXPECT=<<EOF
 a
 b
 a
 b
 EOF
-CMDS=2(foo,?e a,?e b)()
+CMDS=2(foo;?e a;?e b)()
```

Comment:
that wont work on the old parser, you have to quote the whole thign

Reply:
no, i should have fixed this. no need to quote the whole thing because i made it look for `;` only after the `)`.

Reply:
See https://github.com/radareorg/radare2/pull/16179/files#diff-40aa0bc93d40f6d1334c0272c061cafbR2529